lib/chore/duplicate_detector.rb

```diff
@@ -23,7 +14,21 @@ def initialize(opts={})
       @dupe_on_cache_failure = opts.fetch(:dupe_on_cache_failure) { false }
       @timeout               = opts.fetch(:timeout) { 0 }
       @servers               = opts.fetch(:servers) { nil }
-      @memcached_client      = opts.fetch(:memcached_client) { Dalli::Client.new(@servers, memcached_options) }
+      @memcached_client      = opts.fetch(:memcached_client) { memcached_client(memcached_options) }
+    end
+
+    def memcached_client(options={})
```

Comment:
So the idea is that we only complain if you actually attempt to make the call?

Reply:
yea, I just need a way to override the `memcached_client` object _without_ overriding the `Dalli::Client` method (which would require `dalli` in the spec files).